cli/azd/main.go

```diff
@@ -39,7 +40,11 @@ func main() {
 
 	log.SetFlags(log.LstdFlags | log.Lshortfile)
 
-	if !isDebugEnabled() {
+	if isDebugEnabled() {
```

Comment:
?  should this go in?

Reply:
Yes. Without it, we have no debugging story for any Azure requests currently.

Reply:
We should have a different PR for that. With a title like `set log handler for azure sdk client options`

Reply:
Will split